### PR TITLE
feat(config/presets): Add nextra monorepo

### PR DESCRIPTION
**Pull request by danez**
## Changes

Add grouping for nextra monorepo.  https://github.com/shuding/nextra

## Context

I was under the believe that renovate automatically groups by source-URL, but maybe this has changed in the last years.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

The docs are auto-generated?

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/config/presets/internal/monorepo.ts [hunk 1]
```diff
@@ -155,6 +155,7 @@ const repoGroups = {
     'https://github.com/zeit/next.js', // old repo
     'https://github.com/vercel/next.js',
   ],
+  nextra: 'https://github.com/shuding/nextra',
```
Comment (viceice) [inline lib/config/presets/internal/monorepo.ts:158]:
```suggestion
  nextra-js: 'https://github.com/shuding/nextra',
```

I think we should suffix all new monorepos. @rarkins WDYT?

Comment (rarkins) [reply lib/config/presets/internal/monorepo.ts:158]: Is there a non-JS version?

Comment (viceice) [reply lib/config/presets/internal/monorepo.ts:158]: not yet

Comment (rarkins) [reply lib/config/presets/internal/monorepo.ts:158]: I think we can keep it as nextra

**Review by viceice — APPROVED** [review]

**Pull request merged**